splunk-otel-android/src/main/java/io/opentelemetry/rum/internal/SpanFilterBuilder.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.splunk.rum;
+package io.opentelemetry.rum.internal;
```

Comment:
The `SpanFilterBuilder` is part of the public API; we can't just move it to another package without introducing a breaking change.
WDYT about copying the existing `SpanFilterBuilder` over to the otel package and leaving a shim in the splunk package, that'd simply call the otel filter builder methods?

Reply:
Yeah good catch. I missed the fact that it is "api surface".

Reply:
Ok, I copied it back and turned it into a wrapper. Added method `configureSpanFilter` on `SplunkRumBuilder` and deprecated the other method that takes the splunk class. LMK what you think.